2-0-samples.md

```diff
@@ -195,7 +195,9 @@ _Tissue Type_, _Times Received_, _Tube Number_, and _Material_.
 </td></tr></table>
 
 
-Some fields cannot be filled down, so enter each of those separately.
+Some fields cannot be filled down, so enter each of those separately. If you
+have a label with an incrementing number (_e.g._, `PROJ_0001_X`,
+`PROJ_0002_X`), enter two rows, select them, then double click the blue square.
```

Comment:
This might not be the best example to use; I was testing out this functionality for the barcodes of lies pattern and found it wasn't zero-padding them, so it went "BCA01, BCA02, BCA3, BCA4..." so we might want to adjust the documentation to show a non-zero-padded number.